lib/shared/widgets/action_container.dart

```diff
@@ -86,14 +85,20 @@ class _ActionContainerState extends State<ActionContainer> {
             onLongPress: widget.onLongPress == null
                 ? null
                 : () {
-                    if (kDebugMode) print('Long press');
+                    if (widget.onlyLongPress) {
```

Comment:
this if doesnt feel necessary as it only will be executed when onlyLongPress is not null